tests/rpmvfylevel.at

```diff
@@ -346,9 +346,9 @@ nohdrs
 0
 nosig
 /data/RPMS/hello-2.0-1.x86_64-signed.rpm:
-    Header OpenPGP signature: NOTFOUND
-    Header OpenPGP RSA signature: NOTFOUND
-    Header OpenPGP DSA signature: NOTFOUND
+    Header signature: NOTFOUND
```

Comment:
Hmm. These verbose messages are a headache. I'm thinking we should emit "Header OpenPGP signature" for the v6 signatures to have at least the new thing equal in both 4.x and 6.x, although we have to leave the RSA/DSA lines alone to avoid breakage.

Reply:
On a related note, signature/Signature on fail/success is nuts, but probably too risky to change in a stable series :unamused: